chromium_src/chrome/browser/ui/color/material_omnibox_color_mixer.cc

```diff
@@ -0,0 +1,15 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/ui/color/material_omnibox_color_mixer.h"
+
+#define AddMaterialOmniboxColorMixer AddMaterialOmniboxColorMixer_ChromiumImpl
+#include "src/chrome/browser/ui/color/material_omnibox_color_mixer.cc"
```

Comment:
same as other similar overrides there's no reason to include the original file here

Reply:
Fixed in 9cb485879cb12555cdde2652e57aec299bfd3b9a